packages/dapp/src/components/common/tokenImage/TokenImage.jsx

```diff
@@ -0,0 +1,38 @@
+import { Image } from 'components/common/tokenImage/Image';
+import { Video } from 'components/common/tokenImage/Video';
+import React from 'react';
+
+const isVideoToken = ({ address, chainId }) =>
```

Comment:
Does it mean that the video will be displayed only for specific token contract? Since we don't know which NFTs are a video stream is it possible to make decisions based on the tokens metadata rather than on the token contract address?

Reply:
Unfortunately there's no way to know this without fetching the file data, because the zerion token uri returns the video under "image_url". 

Let me see if there's a way to download the file and then check the mimetype before rendering.

Reply:
But yes you're right. currently only zerion NFTs have their image_url rendered as video.